macros/src/main/scala/gestalt/macros/Monadless.scala

```diff
@@ -162,19 +163,13 @@ object Transformer {
           }
 
 
-        case Apply(fun, List(v)) if isUnlift(fun.tpe) => Some(v)
+        case Applysss(fun, _, (v :: Nil) :: Nil) if isUnlift(fun.tpe) => Some(v)
```

Comment:
Indeed, in this case quasiquote pattern is much better, and `$fun` does matter anything, not just identifier. I made the change in latest commit.